docs/ruleset_functions/is_baseline_system_1(HVAC_id).md

```diff
@@ -0,0 +1,35 @@
+# is_baseline_system_1  
+
+**Description:** Get a TRUE or FALSE output as to whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1.  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as system 1 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_baseline_system_1**: The functions returns TRUE if the system is system 1 and FALSE if the system is not system 1.  
+ 
+**Function Call:** 
+1. get_hvac_zone_list_w_area()   
+
+## Logic:  
+- Set the is_baseline_system_1 boolean variable to true (if it does not meet the criteria below then it gets set to false): `is_baseline_system_1 = TRUE`  
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
+- Check that there is only one heating system, one cooling system, that a fan system is defined, and that there is no preheat system associated with the HVAC system. If any of these criteria are not met then the function should return FALSE and no need to carry on with remaining logic: `if Len(hvac_b.heating_system) != 1 OR Len(hvac_b.cooling_system) != 1 AND Len(hvac_b.fan_system) != 1 AND Len(hvac_b.preheat_system) != Null: is_baseline_system_1 = FALSE`  
+- Create an object associate with the heating_system associated with hvac_b: `heating_system_b = hvac_b.heating_system[0]`
+- Check that the data elements associated with the heating_system align with system 1, if not then the function should return FALSE and no need to carry on with remaining logic: `if heating_system_b.heating_system_type != "FLUID_LOOP": is_baseline_system_1 = FALSE`
+- Create an object associate with the cooling_system associated with hvac_b: `cooling_system_b = hvac_b.cooling_system[0]`
```

Comment:
@claperle if heating system is not Fluid Loop, can we skip checking the cooling system?

Reply:
@claperle Good point, yes.

Reply:
@claperle everything in this function looks good. The only additional check might be to make sure the fluid loop is connected to a boiler. I think we have a separate rule that confirms the heating fuel type (correct?) so we might not need to check that. The boiler check is important as for purchased heating we would do everything the same but only check if fluid loop is served by an external fluid source